webui/src/pages/auth/login.jsx

```diff
@@ -51,10 +51,16 @@ const LoginForm = () => {
 
                             <Button variant="primary" type="submit">Login</Button>
                         </Form>
-                        { showResetPwd ?
-                            <Button variant="link" className="reset-pwd" onClick={()=> {router.push("/auth/resetpassword")}}>Reset password</Button>
-                            : ""
-                        }
+                        <div className={"mt-2 mb-1"}>
+                            { showResetPwd ?
+                                <Button variant="link" className={"text-secondary mt-2"}  onClick={()=> {router.push("/auth/resetpassword")}}>Reset password</Button>
+                                : ""
+                            }
+                            { oidcEnabled ?
+                                <Button variant="link" className="text-secondary mt-2" onClick={()=> {window.location = "/oidc/login"}}>Sign in with OIDC provider</Button>
```

Comment:
Consider `s/OIDC Provider/SSO Provider`.  Users don't need to know they're using OIDC, they're just using their GMail.

Reply:
Done